config/locales/requests.en.yml

```diff
@@ -61,7 +61,7 @@ en:
       patron_conf_msg: "On Order books have not yet been received by the library. You have placed a request to be notified when this item has arrived."
       access_msg: 'Access Users are not eligible for On Order requests.'
     paging:
-      email_subject: "Firestone Library Paging Request"
+      email_subject: "Paging Request"
```

Comment:
Users can get the same subject that standard recap request confirmations gets. See previous comment for staff email.